### DISCUSSION TITLE
Can it support lower version?

### DISCUSSION THREAD

**Discussion by uright008** [Impostor/Impostor#566] [Q&A]
I search all of the docs, but I didn't find any word that said can it support lower version or not, if didn't, can it be a new feature in the new version of the Impostor?

**Comment by miniduikboot** [comment]
Sorry, but we don't have the time to support older versions of Among Us. We try our best to keep compatibility with old versions of Among Us but we drop support when newer versions make significant changes to the network code.

If you really need support for old versions of Among Us, I recommend running an old version of Impostor. Please note that the documentation may have changed in the mean time, so if you use an old version I also recommend to use the documentation associated with that version

**Reply by uright008** [reply]
but I can't running any old version

**Reply by miniduikboot** [reply]
I'm not sure I understand your question. Which version of the game are you trying to use?

**Comment by Fluffy728** [comment]
No, we can’t support older versions of among us. Yeah and the team won’t have time to do more hacks. And we are unsure if it actually will work but there would probably be no servers there or unsupported or it crashes.

in case of needing to use old versions of among us, use old versions of imposter and don’t use updates. The docs has changed so use wayback.archive.org to use the docs for different  versions of imposter.

But try an update first.

What device and among us version are you using and I can provide info. If not, mark this as an answer.
